start.ipynb

```diff
@@ -15,6 +15,25 @@
     "}"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "html"
+    }
```

Comment:
I am somewhat surprised that the `nbstripout` pre-commit hook does not remove this. I guess it doesn't matter.